pkg/controllers/management/aks/test/buildakscccreateobject.json

```diff
@@ -53,14 +53,17 @@
     ],
     "podCidr": null,
     "privateCluster": false,
+    "privateDnsZone": null,
```

Comment:
Not a blocking, but just confirming: All the below changes due to https://github.com/rancher/aks-operator/pull/243/files being included in the new operator rc release, as I understand this?